packages/reactivity/src/reactive.ts

```diff
@@ -204,7 +204,7 @@ function createReactiveObject(
   if (existingProxy) {
     return existingProxy
   }
-  // only a whitelist of value types can be observed.
+  // only a free list of value types can be observed.
```

Comment:
```suggestion
  // only an allow-list of value types can be observed.
```
I've seen allow<->block around as a good replacement for white<->black.Is "free list" also common?